src/brogue/Architect.c

```diff
@@ -571,16 +571,15 @@ boolean cellIsFeatureCandidate(short x, short y,
 void addLocationToKey(item *theItem, short x, short y, boolean disposableHere) {
```

Comment:
I'll get around to this at some point - basically I decided to scope this PR to (mostly) just `Items.c` definitions so that it wouldn't just keep ballooning bigger and bigger.